### PR TITLE
converted tests to BDD

### PR DESCRIPTION
**Pull request by duran-thomas**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,9 @@
     "rest": "npx codeceptjs run --grep '@restAPI'",
     "webdriver": "npx codeceptjs run --grep '@webDriver'",
     "janus": "npx codeceptjs run --grep '@JanusBeta'",
-    "prodSafe": "npx codeceptjs run --grep '@prodSafe'"
+    "prodSafe": "npx codeceptjs run --grep '@prodSafe'",
+    "bdd-rest": "npx codeceptjs run --features --grep '@restAPI'",
```
Comment (pegfoltz) [inline package.json:19]: I think you meant to expand @prodSafe to be @productionSafe (as you did in the test)

**Review by pegfoltz — APPROVED** [review]
looks great!  just update your README.md to reflect the @productionSafe tag.  Congratulations!

File: package.json (hunk 1 above)
Comment (duran-thomas) [reply package.json:19]: this is addressed in the latest commit

**Review by pegfoltz — APPROVED** [review]
looks good!

**Pull request merged**